### ISSUE TITLE
Pd style "see also" objects in IM help files would improve ease-of-use.

### ISSUE THREAD

**Issue by alexmlucas** [matthewscharles/instrument-maker#29]
It would be useful to include objects of a similar type in Instrument Maker help files. For instance. im.sinewave, in the im.sawtooth help file and vice-versa.

**Comment by matthewscharles** [comment]
I agree. I left this out because I imagine the default interaction would be to click on them (and load the abstraction).  

I wonder if it's possible to open a Pd file via a message box, and just load up the help files themselves directly.  This might be in the latest version of Pd.

**Comment by matthewscharles** [comment]
I added a test using message boxes to im.acid, but I'm not really happy with it; the patch cables into the new im.help are untidy, and the user needs to lock the patcher (which I generally ask not to do in an introductory session). maybe simply including the objects is the right way around, after all.  if only there were a way to prevent opening an abstraction by clicking on it..

**Comment by alexmlucas** [comment]
I don't mind the message box approach so much. Would a help file not open as locked by default anyway?

**Comment by matthewscharles** [comment]
Good point!

Ok, in that case, I think this could work out nicely.  If you'd like to try it out for yourself, the name of any im object as a message into im.help should load it up.

**Comment by matthewscharles** [comment]
actually I quite like the idea of having a help file for each category in the future (sound sources, inputs, etc). 

this will only work in 0.50 upwards -- that version will also allow us to open web pages. I need to check if this will work on Raspbian, but I'm happy to start going through and adding some suggested objects.

not sure about the layout, hence I haven't added this option to the help templates yet..if you think of anything nicer, or if you have any suggestions for the help files in general, it's open!
